stories/setup/webpack.config.babel.js

```diff
@@ -1,58 +1,18 @@
-import 'core/polyfill';
-
-import path from 'path';
-
-import config from 'config';
-import autoprefixer from 'autoprefixer';
-import webpack from 'webpack';
-
-import { getClientConfig } from 'core/utils';
+import { getPlugins, getRules } from '../../webpack-common';
+import webpackConfig from '../../webpack.prod.config.babel';
 
 module.exports = {
+  ...webpackConfig,
   module: {
     rules: [
-      {
-        test: /\.svg$/,
-        use: [{ loader: 'svg-url-loader', options: { limit: 10000 } }],
-      },
-      {
-        test: /\.scss$/,
-        use: [
-          { loader: 'style-loader' },
-          { loader: 'css-loader', options: { importLoaders: 2 } },
-          {
-            loader: 'postcss-loader',
-            options: {
-              plugins: [
-                autoprefixer({
-                  browsers: ['last 2 versions'],
-                  grid: false,
-                }),
-              ],
-            },
-          },
-          { loader: 'sass-loader', options: { outputStyle: 'expanded' } },
-        ],
-      },
+      ...getRules({ bundleStylesWithJs: true, includeBabelLoader: false }),
     ],
   },
-  // The following plugins are needed to help handle
-  // server-side imported components such as core/logger.
-  // Is this the best way??
+  // TODO: add the circularDependency plugin back once the following issue is fixed:
+  // https://github.com/mozilla/addons-frontend/issues/6561.
   plugins: [
-    new webpack.DefinePlugin({
-      CLIENT_CONFIG: JSON.stringify(getClientConfig(config)),
+    ...getPlugins({
+      includeCircularDependencyPlugin: false,
```

Comment:
I suggest fixing https://github.com/mozilla/addons-frontend/issues/6561 first so that we don't forget to actually do it.